src/flamenco/runtime/fd_executor.c

```diff
@@ -399,14 +399,27 @@ fd_executor_load_transaction_accounts( fd_exec_txn_ctx_t * txn_ctx ) {
 
     /* https://github.com/anza-xyz/agave/blob/v2.0.9/svm/src/account_loader.rs#L304-306 */
     fd_borrowed_account_t * program_account = NULL;
-    int err = fd_txn_borrowed_account_view_idx( txn_ctx, instr->program_id, &program_account );
+    int err = fd_txn_borrowed_account_view_idx_allow_dead( txn_ctx, instr->program_id, &program_account );
     if( FD_UNLIKELY( err!=FD_ACC_MGR_SUCCESS ) ) {
       return FD_RUNTIME_TXN_ERR_PROGRAM_ACCOUNT_NOT_FOUND;
     }
 
     /* https://github.com/anza-xyz/agave/blob/v2.0.9/svm/src/account_loader.rs#L307-309 */
     if( FD_UNLIKELY( !memcmp( program_account->pubkey->key, fd_solana_native_loader_id.key, sizeof(fd_pubkey_t) ) ) ) {
-      continue;
+      /* When program account pubkey is the native loader, Agave sets the program_indices of the LoadedTransaction to [] with
+         no errors until trying to push the specific instruction onto the instruction stack */
+      txn_ctx->null_program_id = 1;
+      return FD_RUNTIME_EXECUTE_SUCCESS;
```

Comment:
does this also happen when there are multiple instructions where NativeLoader is the program id of one of them? returning here seems sus considering that Agave continues to iterate over all of the instructions in the closure.

https://github.com/anza-xyz/agave/blob/ced98f1ebe73f7e9691308afa757323003ff744f/svm/src/account_loader.rs#L297-L300

Reply:
yah, you're right. maybe we need to just have a `program_indices` list as well in the transaction context